skaffold.yaml

```diff
@@ -892,6 +892,7 @@ build:
         - pkg/operation/botanist/matchers
         - pkg/operation/care
         - pkg/operation/common
+        - pkg/operation/common/secrets
```

Comment:
Actually, we wanted to get rid of this "common" package eventually. I would suggest moving the code (including the annotation constants) to
`pkg/utils/gardener/secretsrotation`. WDYT?